src/containers/Extensions/Extensions.module.css

```diff
@@ -5,7 +5,8 @@
 .Dialogbox {
   border-radius: 32px !important;
   margin: 0px;
-  min-width: 456px;
+  min-width: 280px;
+  max-width: 436px;
```

Comment:
Why did we change this? It's looking like this on the desktop screen
![image](https://user-images.githubusercontent.com/32592458/124808896-23d50080-df7d-11eb-9bb1-b9735c6d63ff.png)
 You can use media queries in this case